lm75bd/lm75bd.h

```diff
@@ -5,7 +5,7 @@
 #include <stdint.h>
 
 /* LM75BD I2C Device Address */
-#define LM75BD_OBC_I2C_ADDR /* Define the address here */
+#define LM75BD_OBC_I2C_ADDR 0x4f  /* Define the address here */
```

Comment:
I got confused by this small f for a second because I have seen it used for floats, extreme nit: but use F and then add a U for clarity I guess

Reply:
Fair